<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>390</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Super Hex Table</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="main_layout">
      <item>
       <layout class="QHBoxLayout" name="parent_input_layout">
        <item>
         <layout class="QGridLayout" name="input_layout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Input</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="buffer_preview_menu">
            <item>
             <property name="text">
              <string>Hex</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Decimal</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="input_option_menu">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Buffer preview</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="buffer_preview"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="input_box"/>
          </item>
          <item row="0" column="2">
           <layout class="QHBoxLayout" name="radio_buttons_layout">
            <item>
             <widget class="QRadioButton" name="hex_button">
              <property name="text">
               <string>Hex</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">input_format_group</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="dec_button">
              <property name="text">
               <string>Decimal</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">input_format_group</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="binary_button">
              <property name="text">
               <string>Binary</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">input_format_group</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="text_button">
              <property name="text">
               <string>Text</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">input_format_group</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="settings_button">
            <property name="text">
             <string>Settings</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="BE_LE_layout">
          <item>
           <widget class="QCheckBox" name="group_digits">
            <property name="text">
             <string>Group digits</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="BE_button">
            <property name="text">
             <string>Big Endian</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">BE_LE_group</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="LE_button">
            <property name="text">
             <string>Little Endian</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">BE_LE_group</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="table1_layout_parent">
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="table1_layout">
          <property name="horizontalSpacing">
           <number>6</number>
          </property>
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <item row="0" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>INT8</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Hex</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="float_2"/>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Octal</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="int32"/>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="int64"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>UINT16</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Float</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>INT64</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>UINT64</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="uint8"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="uint64"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>UINT8</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Double</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="int8"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>UINT32</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>INT32</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="hex"/>
          </item>
          <item row="5" column="3">
           <widget class="QLineEdit" name="octal"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="uint32"/>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="double_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>INT16</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="int16"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="uint16"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0" columnstretch="0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Binary</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Text</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="Text"/>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="binary"/>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="encoding_menu">
            <item>
             <property name="text">
              <string>UTF-8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UTF-16 LE</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="input_format_group"/>
  <buttongroup name="BE_LE_group"/>
 </buttongroups>
</ui>
